Firmware/Marlin_main.cpp

```diff
@@ -8850,7 +8873,7 @@ SERIAL_PROTOCOLPGM("\n\n");
     ### M702 - Unload filament <a href="https://reprap.org/wiki/G-code#M702:_Unload_filament">G32: Undock Z Probe sled</a>
     #### Usage
     
-        M702 [ U | C ]
+        M702 [ C ]
```

Comment:
In RepRap gcode wiki it is https://reprap.org/wiki/G-code#M702:_Unload_filament

Do we need to update the wiki for PrusaFimware using `C` for just current?

Reply:
Yes, we should do that. It's a really useful argument for end of print gcodes since you only want to unload the current filament when you have a mmu

Reply:
Updated the wiki